src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -209,6 +211,16 @@ export class FileExplorerComponent implements OnInit {
     return path.substring(1, indexSelectedFolder) + selectedFolder;
   }
 
+  dataFinishedLoading() {
+    let dataFinishedLoading = false;
+    if (this.isHome && this.firecloudService.getIsResponseComplete()) {
```

Comment:
Is it necessary to check `this.isHome` here? I would expect this function to look like:
```
dataFinishedLoading() {
  return !(this.firecloudService.getIsResponseComplete() || this.bucketService.getIsResponseComplete());
}
```